README.md

```diff
@@ -32,13 +43,14 @@ To get a local copy up and running follow these simple example steps.
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/mahdimahdawi).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] The MIT License link should be an individual link to your MIT file in your project. It's not a requirement to have one, but make sure that if you add that to your project and README file, your README links to the correct file in your repository.
 - [OPTIONAL] The note you add at the bottom may not be necessary since your README file is not a template so another user can copy and understand the MIT license recomendation.

Reply:
What link can i add here